src/SmartComponents/CompletedTaskDetails/CompletedTaskDetails.js

```diff
@@ -69,7 +69,10 @@ const CompletedTaskDetails = () => {
       if (isError(taskDetails)) {
         setErrors(taskDetails);
       } else {
-        const taskJobs = await fetchExecutedTaskJobs(id);
+        const findCount = await fetchExecutedTaskJobs(id);
+        const count = findCount.meta.count;
+        const path = `?limit=${count}&offset=0`;
```

Comment:
Hm what if you just set the limit to very large number (in Vulnerability we use `2^31-1`) instead of fetching the count and then fetching the exact number of systems?